meta-sokol-flex-distro/recipes-core/packagegroups/packagegroup-cross-canadian.bbappend

```diff
@@ -0,0 +1 @@
+RDEPENDS:${PN}:tcmode-external:sokol-flex = "meta-environment-${MACHINE}"
```

Comment:
How do we ship the toolchain in SDKs then? I guess I am not following this correctly.

Reply:
That's not implemented yet for external toolchains. We'd drop this when we implement external toolchain redistribution.

Reply:
This was needed to prevent building and including an internal toolchain with a built Yocto SDK/ADE while building using an external toolchain. That is, we expected a user using the external codebench toolchain to build Flex would also want an app developer to use a codebench toolchain, not a Yocto-built internal one.